core/src/main/java/org/hibernate/ogm/type/impl/ByteMappedType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.type.impl;
+
+import org.hibernate.MappingException;
+import org.hibernate.engine.spi.Mapping;
+import org.hibernate.ogm.type.descriptor.impl.ByteMappedGridTypeDescriptor;
+import org.hibernate.type.descriptor.java.ByteTypeDescriptor;
+
+/**
+ * @author Davide D'Alto
+ */
+public class ByteMappedType extends AbstractGenericBasicType<Byte> {
```

Comment:
Ok, but what about my question. What kind of consequences will we have if we have multiple types registering the same name? Should we add some protection against that?